eland/field_mappings.py

```diff
@@ -102,9 +102,13 @@ def is_es_agg_compatible(self, es_agg) -> bool:
         # Except "median_absolute_deviation" which doesn't support bool
         if es_agg == "median_absolute_deviation" and self.is_bool:
             return False
-        # Cardinality and Count work for all types
+        # Cardinality, Count and mode work for all types
         # Numerics and bools work for all aggs
-        if es_agg in ("cardinality", "value_count") or self.is_numeric or self.is_bool:
+        if (
+            es_agg in ("cardinality", "value_count", "mode")
```

Comment:
Change this to a set `{"cardinality", "value_count", "mode"}` instead of tuple now that 3+ elements

Reply:
Yep